packages/api/seeders/20200909202338-userseed.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
           id: 'bc3cd395-226c-46be-9b80-89c5f1c944db',
           email: 'bill@email.com',
           username: 'BillsAccount',
-          password: 'asDf',
+          // password: 'asDf',
```

Comment:
Can we undo this?

Reply:
When I ran seeder files the seeders I received an error of "password" not found or doesn't exist. 
Then commented it out to see if seeders ran successfully and it did. 
Should it be removed?

Reply:
Yeah, if password is no longer a valid field, we should delete this line.